mqtt/mqtt-bridge/src/persist/mod.rs

```diff
@@ -5,17 +5,17 @@ use futures_util::stream::Stream;
 use mqtt3::proto::Publication;
 use parking_lot::Mutex;
 
-mod memory;
+pub mod memory;
 
 /// Persistence used in bridge.
 /// Elements are added, then can be removed once retrieved by the loader
 /// If one attempts to remove added elements before reading via the loader remove returns None
 #[async_trait]
-trait Persist<'a> {
+pub trait Persist<'a> {
```

Comment:
I don't see lifetime `'a` used anywhere. Should we remove that?

Reply:
same as above, so will just leave it as is for now